lib/events.js

```diff
@@ -81,7 +90,7 @@ const lazyDOMException = hideStackFrames((message, name) => {
 
 
 function EventEmitter(opts) {
-  EventEmitter.init.call(this, opts);
+  ReflectApply(EventEmitter.init, this, arguments);
```

Comment:
This does not have the same behaviour. Why don't you use FunctionPrototypeCall here and in other places?

Reply:
My intuition was using `arguments` might be more efficient (that's what Babel is using when creating subclasses), but that seems to wrong indeed. I'm reverting that to `FunctionPrototypeCall`.